vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi_test.go

```diff
@@ -60,8 +60,8 @@ func assertEqual(t *testing.T, expected, actual interface{}) {
 
 func TestGetDefinitionName(t *testing.T) {
 	testType := TestType{}
-	typePkgName := "k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/endpoints/openapi.TestType"
```

Comment:
hopefully this only happens during unit test runs.  Even if openapi is broken, I think we should merge the rebase.